src/styles/components/_presentation.scss

```diff
@@ -164,15 +160,15 @@
   text-overflow: ellipsis;
 }
 
-.presentation .thumbnail .caption .size {
+.presentation .caption .size {
```

Comment:
Did you test this change? Removing tumbnail might select something else? Probably best not removed.